### PR TITLE
Send Message txns to SQS Queue

### PR DESCRIPTION
**Pull request by lgbello**
- Rename xNotification to xTransaction
- Add Message txn handling

### CODE REVIEW

**Comment by piratekev** [comment]
run `gofmt -w lib/sqs.go`

**Review by chasewm — APPROVED** [review]

**Comment by piratekev** [comment]
Here's the workflow for testing changes:

1. Merge in changes to core
2. update backend github build script to push images to dev ecr built from pull requests - [example commit](https://github.com/TheBitgram/backend/commit/5a73b211329298c1951d9ecf4bb94901d71ea7cf)
3. open a PR against backend with changes
  i. Wait for the build to complete successfully & find the pr-xxx image in ECR
5. update the backend dev main.tf image version with the PR version in my local
6. run TF apply in dev
  i. while TF apply is running - kill the existing backend pod (there are not enough resources to run 2 backends - plus there’s only one volume that can’t be shared)
8. Make sure the new node starts up and runs as expected
9. Test the dev node
10. cleanup
  i. Replace the version in dev main.tf
  ii. run TF apply in dev
    a. while TF apply is running - kill the existing backend pod (there are not enough resources to run 2 backends - plus there’s only one volume that can’t be shared)
  iii. Revert any bad changes made to core

**Comment by piratekev** [comment]
Feel free to merge once `go-fmt` check passes

File: lib/sqs.go [hunk 1]
```diff
@@ -134,10 +146,22 @@ func (sqsQueue *SQSQueue) SendSQSTxnMessage(mempoolTxn *MempoolTx) {
...
 }
 
-func makeSubmitPostNotification(mempoolTxn *MempoolTx) *SubmitPostNotification {
+func makePrivateMessageNotification(mempoolTxn *MempoolTx) *PrivateMessageTransaction {
+	metadata := mempoolTxn.Tx.TxnMeta.(*PrivateMessageMetadata)
+	affectedPublicKeys := mempoolTxn.TxMeta.AffectedPublicKeys
+	return &PrivateMessageTransaction{
+		AffectedPublicKeys:             affectedPublicKeys,
+		TransactorPublicKeyBase58Check: mempoolTxn.TxMeta.TransactorPublicKeyBase58Check,
+		RecipientPublicKey:             string(metadata.RecipientPublicKey),
```
Comment (summraznboi) [inline lib/sqs.go:155]: using string is not a good idea, since these bytes are random bytes - you probably want base58 or at least hex

Comment (summraznboi) [reply lib/sqs.go:155]: Also, FYI that recipient is the messaging key, so downstream service should be aware of this.

**Pull request merged**